Automattic/jetpack
# Actions: Playground for learning how to get GH context and use conditionals

Fixes nothing. I just want to see how [contexts and expressions](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/contexts-and-expression-syntax-for-github-actions) work in GitHub actions. 

#### Changes proposed in this Pull Request:

NA 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
no

#### Testing instructions:






Push to this branch with whatever testing you want to do. Visit actions to see what happened. 

#### Proposed changelog entry for your changes:

NA
